packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -45,9 +44,26 @@ export function mergeStats(...stats: ISummaryStats[]): ISummaryStats {
     return results;
 }
 
+function byteLength(str: string): number {
```

Comment:
I think name of the function should reflect the fact that byte length is for utf8 encoding of a given string, not some other encoding